### DISCUSSION TITLE
newborns and adolescents

### DISCUSSION THREAD

**Discussion by yangxiaoYY** [MIT-LCP/mimic-code#1559] [MIMIC-IV]
I am a pediatrician who recently gained access to your database for a study that aims to utilize the large-scale population cohort data in MIMIC. Our research plans involve the utilization of advanced statistical methods and causal inference techniques to systematically analyze the association and causal effects of maternal-related diseases on offspring disorders. Additionally, we aim to explore possible mechanisms and intervention measures.
I have a few questions regarding the dataset, and I would be grateful if you could provide some insights:
1. Release Date of MIMIC-IV Newborn and Adolescent Data:
   I noticed that the newest version of MIMIC-IV does not include data specifically pertaining to newborns and adolescents. Could you kindly inform me of the approximate release date for a collection that incorporates this data?
2. Availability of Previous Versions:
   Is there an older version of MIMIC, preferably MIMIC-III, that includes data related to newborns and adolescents? If so, could you confirm whether this dataset is still accessible and usable for research purposes?
3. Age Values below 1 in MIMIC-IV 1.0:
   While exploring the MIMIC-IV 1.0 dataset, I observed that the age of some participants, after extraction, appears to be less than 1. I would appreciate it if you could clarify what this age representation signifies.

**Comment by karinfridkin** [comment]
Did you find out the answer? I'm currently in a similar problem.
